tools/nxstyle.c

```diff
@@ -719,6 +723,63 @@ static void show_usage(char *progname, int exitcode, char *what)
   exit(exitcode);
 }
 
+#ifndef HAVE_STRNDUP
+/********************************************************************************
+ * Name: my_strndup
+ *
+ * Description:
+ *   Duplicate a specific number of bytes from a string.
+ *   Implementation of strndup() for Windows Native
+ *   MinGW does not seem to provide strndup
+ ********************************************************************************/
```

Comment:
```suggestion
 *   MinGW does not seem to provide strndup
 ********************************************************************************/
 
```

look at https://github.com/apache/nuttx/issues/12133